lib/Zim.js

```diff
@@ -99,6 +100,24 @@ Zim.prototype.computeZimName = function() {
     return (this.publisher ? this.publisher.toLowerCase() + '.' : '') + this.env.computeFilenameRadical(false, true, true);
 };
 
+Zim.prototype.computeZimTags = function() {
+    var tags = [];
+
+    /* Mediawiki hostname radical */
+    var mwUrlHostParts = urlParser.parse(this.env.mw.base).host.split(".");
+    var mwUrlHostPartsTag = mwUrlHostParts.length > 1 ?
+	mwUrlHostParts[mwUrlHostParts.length - 2] : mwUrlHostParts[mwUrlHostParts.length - 1]
+    tags.push(mwUrlHostPartsTag.toLowerCase());
```

Comment:
https://github.com/openzim/mwoffliner/issues/147#issuecomment-350498365 wanted first letter to be uppercased ... not use how PhET will be generated however.

Reply:
Yes, I do not see how to do that everywhere, but I have created a new --customZimTags option which goes in that direction if we really want to do that in the future.